### DISCUSSION TITLE
"Deadline Exceeded" while connecting to Milvus via pymilvus.

### DISCUSSION THREAD

**Discussion by Yagyansh** [milvus-io/milvus#4922] [Q&A and General discussion]
Hi. I am using Milvus version 0.10.0 and pymilvus version 0.2.15. I have been running via and search pipelines via this setup for around 3 months now. 
Suddenly, for the past 3-4 days, I am receiving timeouts from Milvus while connecting via pymilvus. This is very strange to be happening on an existing setup. This is impacting my prod Deduplication pipelines severely. Can someone please help identify the issue?

`milvus.client.exceptions.NotConnectError:` Cannot check server version: Error occurred. Deadline Exceeded\n", "filename": "glogging.py", "lineno": 268}`

**Comment by yhmo** [comment]
Sorry for late reply. Is any error log under the /milvus/logs ?
If you run milvus by docker container, you can use 'docker exec -it [container id] bash' to enter the container and find log file under /var/lib/milvus/logs

**Reply by Yagyansh** [reply]
Nope. There is no error in the logs which makes it more worrisome. Also, this is not continuous. If I restart Milvus, this error will go for a while and pymilvus would seamlessly connect to Milvus, but after a brief period, it will start throwing these errors again.
